src/arm/arm.c

```diff
@@ -98,7 +98,9 @@ mraa_arm_platform()
             platform_type = MRAA_96BOARDS;
         else if (mraa_file_contains("/proc/device-tree/model", "ROCK960"))
             platform_type = MRAA_96BOARDS;
-        else if (mraa_file_contains("/proc/device-tree/model", "ZynqMP ZCU100 RevC"))
+       else if (mraa_file_contains("/proc/device-tree/model", "Avnet Ultra96 Rev1"))
```

Comment:
Can you please move this below `ZynqMP ZCU100 RevC`? I'd like to maintain the hierarchy here.

Reply:
In addition, the line seems to be off by one space - please check this one as well.

Reply:
Resolved in current patch version - thanks!